diagnostic_tool/diagnostic_script.py

```diff
@@ -0,0 +1,165 @@
+import sys
+import blosc
+import dill
+import numpy as np
+from time import sleep, time
+from functools import partial
+from interbotix_xs_modules.xs_robot.arm import InterbotixManipulatorXS
+
+
+if sys.argv[1] == "deploy":
+    DEPLOY_MODE = True
+else:
+    DEPLOY_MODE = False
+
+
+class WX250sManipulator:
+    def __init__(self):
+        self.bot = InterbotixManipulatorXS("wx250s", "arm", "gripper")
+        self.arm = self.bot.arm
+        self.joint_names = self.arm.group_info.joint_names
+        self.num_joints = len(self.joint_names)
+
+    def get_gripper_tf(self):
+        return self.arm.T_sb
```

Comment:
I know that this is part of interbotix's code, but what does `sb` mean?

Reply:
Transformation from `body` frame to `space` frame. `Body` refers to the gripper base and `space` refers to the end effector.